Specs/Scene/PostProcessStageLibrarySpec.js

```diff
@@ -119,7 +119,8 @@ describe(
       });
     });
 
-    it("per-feature black and white", function () {
+    // glTF 1.0 model needs to be converted to 2.0
```

Comment:
Can this be replaced with a glTF 2.0 box?

Reply:
Possibly, let me see if I can drop in `BoxTextured` (I don't think we have just a 2.0 box). If not, I'll update this when I work on cleaning up glTF 1.0 assets

Reply:
well, I got one more spec to work, but the others will take more thought, probably have to adjust the camera setup.